### ISSUE TITLE
outputPath not working

### ISSUE THREAD

**Issue by legendar** [webpack-contrib/file-loader#108]
file-loader 0.9.0
webpack 1.13.1
node 6.9.1

my config is:
```
      {
        test: /\.woff2?$|\.ttf$|\.eot$/,
        loader: 'file-loader',
        query: {
          publicPath: 'static/fonts',
          outputPath: 'static/fonts',
          name: '[hash].[ext]',
        }
      }
```

**What is the current behavior?**
files saved directly to main `output.path`

**What is the expected behavior?**
files saved in `output.path`/`outputPath`

**Workaround**
place path into `name` option
```
name: 'static/fonts/[hash].[ext]',
```

**Comment by ekoneko** [comment]
Me too, 

I found the 0.9 version in github and npm are not same.

The npm version don't have the `options.outputPath`

**Comment by nickjanssen** [comment]
Confirmed not working, thanks for the workaround

**Comment by JalAzimi** [comment]
@legendar : It seems that some features on GitHub version are not yet deployed to npm registry.
Try installing from GitHub repo: `npm install webpack/file-loader --save-dev`. You also have to use trailing slash on your paths and it should work:
```javascript
        query: {
          publicPath: 'static/fonts/',
          outputPath: 'static/fonts/',
          name: '[hash].[ext]',
        }
```

**Comment by roastlechon** [comment]
Related to #106

**Comment by gravitycode** [comment]
@legendar This workaround doesn`t work when you have entries like 'app/bundle' 'manual/bundle'. I use this trick to isolate bundles in the distribution. If the path is in the name the bundles it will try to load the assets in manual/assets/image.img on order to /assets/image.img 

dist/
   app/
   manual/
   assets/

I hope that with publicPath and outputPath working i can solve my problem.

**Comment by Tiendq** [comment]
+1 confirmed that it doesn't work with current `file-loader` version `0.9.0`

My test cases:

```
// Main webpack output section
output: {
  path: "./static/",
  publicPath: "/static/"
}
```

1. Test 1
Doesn't work, `img.src` is `/static/images/name.txt` but actual output is `/static/name.txt`

```
{
  test: /\.(png|jpg|svg)$/,
  loader: 'file',
  query: {
  name: '[name].[hash].[ext]',
  publicPath: '/static/images/'
}
```

2. Test 2
Doesn't work too, `img.src` is `/static/images/name.txt` and actual output is still `/static/name.txt`. loader's `outputPath` takes no effect on main `output.path` or `output.publicPath`

```
{
  query: {
  name: '[name].[hash].[ext]',
  publicPath: '/static/images/',
  outputPath: '/static/images/'
}
```

3. Test 3 (@legendar workaround)
Work. `img.src` is correct as `/static/images/name.txt`, image shown.

```
{
  query: {
  name: 'images/[name].[hash].[ext]'
}
```

**Comment by yhtml5** [comment]
Thank you for solving my problem. It works
@JalAzimi

**Comment by Tiendq** [comment]
I just found @JalAzimi tip works amazingly. The `npm` version of `file-loader` doesn't have a `outputPath` check like `github` version. Here is my updated configuration, which I got script bundles in `/static/scripts` and images in `/static/images`:

```javascript
{
  loaders: [{
    test: /\.(png|jpg|svg)$/,
    loader: 'file',
    query: {
      name: '[name].[hash:7].[ext]',
      publicPath: '/static/images/',
      outputPath: '../images/'
    }
  }]
},
output: {
  path: "./static/scripts",
  publicPath: "/static/scripts"
}
```

**Comment by JalAzimi** [comment]
@yhtml5 @Tiendq But you should be careful about your project dependencies and switch to a stable version when it's released with the features implemented.

**Comment by p4bloch** [comment]
This just got released as `0.10.0` 🎉
